### PR TITLE
Add keyboard navigation support.

### LINKED ISSUE #24: Menu (small screen) and Cart side drawer content not accessible by keyboard.
User should be able to tab through menu items or links in cart drawer.

### PR DESCRIPTION
**Pull request by g1st**
This would fix #24. One issue, though. Navigation drawer on small screen uses material-ui <List> component so user can navigate with `Tab` key. On the other hand, in navigation bar Works' dropdown uses \<Menu> component and it can be navigated only with arrow keys. A little inconsistency but maybe we can live with that?

### CODE REVIEW

File: components/NavBar.js [hunk 1]
```diff
@@ -184,7 +184,7 @@ class NavBar extends React.Component {
                   id="simple-menu"
                   anchorEl={anchorEl}
                   open={Boolean(anchorEl)}
-                  onClose={this.handleClose}
+                  onClose={e => this.handleClose(e, 'backdropClick')}
```
Comment (nikrb) [inline components/NavBar.js]: isn't this bad practice? something about creating a new function every render?
As `<Menu/>` is material-ui could we use different functions? `handleBackdropClick`, `handleWorksClick` ?
is that right?

**Review by nikrb — DISMISSED** [review]
LGTM

File: components/NavBar.js (hunk 1 above)
Comment (g1st) [reply components/NavBar.js]: Hmm, I'm not sure but isn't it quite common practice to pass function to react event handlers to have event reference?

Comment (g1st) [reply components/NavBar.js]: Ok, you are ~~probably~~ totally right it is creating new function on every rendering. Still investigating :) I think it is no a big issue but I've sorted it out in last commit hopefully.

File: components/NavBar.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ class NavBar extends React.Component {
     this.setState({ anchorEl: event.currentTarget });
   };
 
-  handleMenuItemClick = (href, as) => {
+  handleMenuItemClick = (href, as) => () => {
```
Comment (g1st) [inline components/NavBar.js:100]: I think this does the job but I still can't wrap my head around it.

**Review by nikrb — DISMISSED** [review]
LGTM

**Review by nikrb — APPROVED** [review]
Hopefully the function creation on render won't cause an issue in the immediate future.
LGTM

**Pull request merged**